Terminal.Gui/ConsoleDrivers/FakeDriver/FakeConsole.cs

```diff
@@ -798,7 +798,7 @@ public static ConsoleKeyInfo ReadKey (bool intercept)
 			if (MockKeyPresses.Count > 0) {
 				return MockKeyPresses.Pop();
 			} else {
-				return new ConsoleKeyInfo ('~', ConsoleKey.Oem3, false,false,false);
```

Comment:
IIRC, I choose `~` over `\0` because it can be typed and will never be mistaken for a string terminator.

I think going with `\0` will likely work fine, but I don't understand why you think you need to change it. If you do change it, you need to change the `FakeDriver` tests as well (see other comment below) because they use `~`.

Reply:
I want to iterate to keys events and and capture all that is being sending by `MockKeyPresses` or by `SendKeys`. With the `'\0'` the events won't be called and I can capture all with cleaning way.